packages/flutter/test/widgets/fade_in_image_test.dart

```diff
@@ -262,6 +265,44 @@ Future<void> main() async {
       expect(findFadeInImage(tester).target.opacity, moreOrLessEquals(1));
     });
 
+    testWidgets('memory placeholder cacheWidth and cacheHeight is passed through', (WidgetTester tester) async {
+      final Uint8List testBytes = Uint8List.fromList(kTransparentImage);
+      final FadeInImage image = FadeInImage.memoryNetwork(
+        placeholder: testBytes,
+        image: 'test.com',
+        placeholderCacheWidth: 20,
+        placeholderCacheHeight: 30,
+        imageCacheWidth: 40,
+        imageCacheHeight: 50,
+      );
+
+      final DecoderCallback decode = (Uint8List bytes, {int cacheWidth, int cacheHeight}) {
+        expect(cacheWidth, 20);
+        expect(cacheHeight, 30);
+        return PaintingBinding.instance.instantiateImageCodec(bytes, cacheWidth: cacheWidth, cacheHeight: cacheHeight);
+      };
+      final ResizeImage resizeImage = image.placeholder;
+      resizeImage.load(await resizeImage.obtainKey(ImageConfiguration.empty), decode);
```

Comment:
It makes me a bit uncomfortable that the test doesn't verify that the decode callback was actually invoked. If for some reason it doesn't run, the expects in there also wouldn't execute.

Reply:
Verifies it is called now!